### PR TITLE
Reducers and actions

### PR DESCRIPTION
**Pull request by shahnawaza75**
Configured Redux Store.
Added reducer for adding and removing books.

### CODE REVIEW

File: src/redux/books/Books.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  type: ADDBOOK,
+  book,
+
+});
+
+export const removing = (book) => ({
+  type: REMOVE,
+  book,
+});
```
Comment (rotimiazeez) [inline src/redux/books/Books.js]: - Please note that I did not get a new book object passed by action.payload when I try adding a new book. 😞

File: src/redux/categories/Categories.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const checkReducer = (state = categoriesState, { type }) => {
+  switch (type) {
+    case CHECK_STATUS:
+      return { categories: ['Under construction...'] };
+    default:
+      return state;
+  }
+};
+
```
Comment (rotimiazeez) [inline src/redux/categories/Categories.js]: - Kindly make sure when you click the check status button, it returns a string "Under construction" on your page.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @shahnawaza75  👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯 
- Good readme file
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @shahnawaza75 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**